kolibri_exercise_perseus_plugin/package.json

```diff
@@ -6,9 +6,10 @@
   "description": "Exercise/Perseus Render Plugin for Kolibri",
   "private": true,
   "dependencies": {
+    "babel-polyfill": "^6.23.0",
     "btoa": "1.1.2",
     "jquery": "2.2.4",
-    "qtip2": "2.20",
+    "qtip2": "2.2.0",
```

Comment:
Hope this was OK. `yarn install` kept asking me to specify the version in the command line